src/m365/pp/commands/dataverse/dataverse-webresource-list.ts

```diff
@@ -0,0 +1,201 @@
+    'contentjson',
+    'contentjsonfileref',
+    'createdon',
+    'dependencyxml',
+    'description',
+    'displayname',
+    'introducedversion',
+    'isavailableformobileoffline',
+    'iscustomizable',
+    'isenabledformobileclient',
+    'ishidden',
+    'ismanaged',
+    'languagecode',
+    'modifiedon',
+    'overwritetime',
+    'silverlightversion',
+    'solutionid',
+    'versionnumber',
+    'webresourceidunique',
+    'webresourcetype'
+  ];
+
+  public async commandAction(logger: Logger, args: CommandArgs): Promise<void> {
+    await this.logIfVerbose(logger, `Retrieving list of web resources for solution '${args.options.solutionId}' in environment '${args.options.environmentName}'...`);
+
+    try {
+      const dynamicsApiUrl = await powerPlatform.getDynamicsInstanceApiUrl(args.options.environmentName);
+
+      this.logIfVerbose(logger, `Found Dynamics API URL: ${dynamicsApiUrl}`);
+
+      if (await this.checkIfSolutionExists(args.options.solutionId, dynamicsApiUrl, logger) === false) {
+        throw new Error(`Solution with ID '${args.options.solutionId}' not found in environment '${args.options.environmentName}'`);
+      }
+
+      const solutionComponentsRequestUrl = `${dynamicsApiUrl}/api/data/v9.2/solutioncomponents?$filter=_solutionid_value eq ${args.options.solutionId} and componenttype eq 61&$select=objectid`;
+
+      this.logIfVerbose(logger, `Retrieving web resources solution components`);
+      const solutionComponentResponse = await odata.getAllItems<SolutionComponent>(solutionComponentsRequestUrl);
+
+      this.logIfVerbose(logger, `Found ${solutionComponentResponse.length} web resource solution component in solution '${args.options.solutionId}' in environment '${args.options.environmentName}'`);
+      const props = this.getWebresourceProperties(args);
+
+      this.logIfVerbose(logger, `Using props: ${props.join(',')}`);
+      const webResourcesRequestUrl = `${dynamicsApiUrl}/api/data/v9.2/webresourceset?$select=${props.join(',')}&$filter=${solutionComponentResponse.map((c: SolutionComponent) => `webresourceid eq ${c.objectid}`).join(' or ')}`;
+
+      this.logIfVerbose(logger, `Retrieving web resources from URL: ${webResourcesRequestUrl}`);
+      const webResourcesResponse = await odata.getAllItems<Webresource>(webResourcesRequestUrl);
+
+      if (!args.options.output || !cli.shouldTrimOutput(args.options.output)) {
+        const outputWithLabels = webResourcesResponse.map(webresource => {
+          return {
+            ...webresource,
+            webresourceTypeLabel: WEBRESOURCE_TYPE_LABELS[webresource.webresourcetype - 1],
+            componentStateLabel: COMPONENT_STATE_LABELS[webresource.componentstate],
+            isManagedLabel: webresource.ismanaged ? 'Managed' : 'Unmanaged',
+            isEnabledForMobileClientLabel: webresource.isenabledformobileclient ? 'Yes' : 'No',
+            isAvailableForMobileOfflineLabel: webresource.isavailableformobileoffline ? 'Yes' : 'No'
```

Comment:
I think you can remove these properties. According to me, a boolean with `true`, `false` is clear enough here.